ports/powerpc/README.md

```diff
@@ -0,0 +1,40 @@
+# The minimal port
```

Comment:
should be something like "The PowerPC port"

Reply:
yep